extensions/Bitter_130/SQLSnapFileshare.js

```diff
@@ -0,0 +1,160 @@
+// By: Bitter_130
+
+(function (Scratch) {
+  "use strict";
+
+  class SQLSnapFileshare {
+    constructor() {
+      this.serverURL =
+        "https://snapextensions.uni-goettingen.de/handleTextfile.php";
+    }
+
+    getInfo() {
+      return {
+        id: "bitter130SQLSnapFileshare",
+        name: "SQLSnap! " + Scratch.translate("Fileshare"),
+        color1: "#31b3d4",
+        color2: "#179fc2",
+        blocks: [
+          {
+            opcode: "setServerURL",
+            blockType: Scratch.BlockType.COMMAND,
+            text: Scratch.translate("set server URL to [URL]"),
+            arguments: {
+              URL: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: this.serverURL,
+              },
+            },
+          },
+          {
+            opcode: "saveToServer",
+            blockType: Scratch.BlockType.COMMAND,
+            text: Scratch.translate("save [FILE] with content [CONTENT]"),
+            arguments: {
+              FILE: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: Scratch.translate("data") + ".txt",
+              },
+              CONTENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: Scratch.translate("Hello World!"),
+              },
+            },
+          },
+          {
+            opcode: "loadFromServer",
+            blockType: Scratch.BlockType.REPORTER,
+            text: Scratch.translate("load [FILE]"),
+            arguments: {
+              FILE: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: Scratch.translate("data") + ".txt",
+              },
+            },
+          },
+          {
+            opcode: "fits",
+            blockType: Scratch.BlockType.BOOLEAN,
+            text: Scratch.translate("[DATA] fits on server?"),
+            arguments: {
+              DATA: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: Scratch.translate("data"),
+              },
+            },
+          },
+          {
+            opcode: "deleteFromServer",
+            blockType: Scratch.BlockType.COMMAND,
+            text: Scratch.translate("delete [FILE]"),
+            arguments: {
+              FILE: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: Scratch.translate("data") + ".txt",
+              },
+            },
+          },
+        ],
+      };
+    }
+
+    setServerURL(args) {
+      args.URL = Scratch.Cast.toString(args.URL);
+      this.serverURL = args.URL;
+    }
+
+    saveToServer(args) {
+      args.FILE = Scratch.Cast.toString(args.FILE);
+      args.CONTENT = Scratch.Cast.toString(args.CONTENT);
+
+      const url =
+        this.serverURL +
+        "?type=write" +
+        "&content=" +
+        encodeURIComponent(args.CONTENT) +
+        "&filename=./textfiles/" +
+        encodeURIComponent(args.FILE);
+
+      return fetch(url)
```

Comment:
I know lint already threw this as an error, but for security reasons, please use Scratch.fetch() instead of fetch()

Reply:
*this goes for the 2 other times fetch() is used as well

Reply:
I seemed to have missed this. Late night PR management does NOT treat you well 😅

Reply:
@namelessisbackbutnottodobadthingsiswear Alright, check it out. Thank you so much for pointing out what I missed. I'm so tired 😪